ReactViewResources/Loader/Public/ViewFrame.tsx

```diff
@@ -93,15 +96,29 @@ class InternalViewFrame<T> extends React.Component<IInternalViewFrameProps<T>, {
         childView.generation = this.generation;
         childView.parentView = this.parentView;
         childView.context = this.props.context;
-
-        const loadedHandler = this.props.loaded;
-        if (loadedHandler) {
-            childView.viewLoadTask.promise.then(() => loadedHandler());
```

Comment:
we were triggering the loaded event when the inner content has finished loading.
The idea now is to do so when all inner views finish loading.
So that, on the Workspace.view in ServiceStudio we can listen to the event

Reply:
Why not add a different event for when the InnerViews load has completed? (I'm questioning, because maybe both events are valid)

Reply:
not sure what was the intention of this event on the first place
@joaompneves

Reply:
but if you have multiple inner views nested on a dialog, you would only listen to the event of the first loaded one